Source/com/drew/metadata/exif/ExifTiffHandler.java

```diff
@@ -59,7 +60,13 @@
 {
     public ExifTiffHandler(@NotNull Metadata metadata, @Nullable Directory parentDirectory)
```

Comment:
This constructor cannot be removed, because it is also used by Png, Tiff and Eps, which are out of scope of this pull request. Eventually it can be removed.